res/database/Default/database_programmes.xml

```diff
@@ -5492,8 +5594,12 @@ New Year's Eve fun and ritual to watch in some countries.</en>
 				<member index="3" function="2">02fb6aa3-5447-4dbd-aecd-3f2dc510c4a9</member>
 			</staff>
 			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
-			<data country="D" year="1963" distribution="1" maingenre="5" subgenre="" flags="8" blocks="1" price_mod="1" />
-			<ratings critics="37" speed="70" outcome="70" />
+			<data country="D" year="1963" distribution="2" maingenre="5" subgenre="" flags="8" blocks="1" price_mod="0.25" />
+			<ratings critics="70" speed="70" outcome="70" />
+			<modifiers>
+				<modifier name="topicality::age" value="0.25" />
+				<modifier name="topicality::refresh" value="0.1" />
```

Comment:
refresh 0.1 will definitely not work well with the balancing changes made

Reply:
What would be the correct value for taking approx. 12 months to recover?

Reply:
I haven't had time to test-play. Are you able to compile TVTower yourself? If so, try out the database with the current master. If not, I don't know what the status of GitHub-actions is (can we build a snapshot version here).
Please keep this point as one of the most important to change - many of the others are more a matter of taste.

Reply:
out-commented the line for now / changed to TODO